ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/EthHash.java

```diff
@@ -71,12 +74,12 @@
    * @return A byte array holding MixHash in its first 32 bytes and the EthHash result in the in
    *     bytes 32 to 63
    */
-  public static byte[] hashimotoLight(
+  public static PoWSolution hashimotoLight(
       final long size, final int[] cache, final byte[] header, final long nonce) {
```

Comment:
the return got wrapped into a PoWSolution that wrapped the byte array in `Bytes`, why is the input still a byte array?

Reply:
sure